pkg/plugins/client.go

```diff
@@ -99,9 +99,9 @@ func (c *Client) callWithRetry(serviceMethod string, args interface{}, ret inter
 		if resp.StatusCode != http.StatusOK {
 			remoteErr, err := ioutil.ReadAll(resp.Body)
 			if err != nil {
-				return &remoteError{err.Error(), serviceMethod}
+				return remoteError{err.Error(), serviceMethod}
 			}
-			return &remoteError{string(remoteErr), serviceMethod}
+			return remoteError{string(remoteErr), serviceMethod}
```

Comment:
Oh yes, I totally cross-eyed this and saw the err.Error() in red on the left, and the green on the right thinking it was changed.